nvtabular/ops/categorify.py

```diff
@@ -285,6 +287,9 @@ def __init__(
                 "with this num_buckets setting!"
             )
 
+        self.vocabs = self.process_vocabs(vocabs if vocabs is not None else {})
```

Comment:
Would simplify the calling code a bit if the check for `None` and associated default happened at the top of the `process_vocabs` method